.github/workflows/preview-publish-ga.yml

```diff
@@ -63,7 +68,9 @@ jobs:
 
   biojava:
     runs-on: ubuntu-latest
-    if: github.repository == 'Kotlin/dokka'
+    if: >  # Run this job only for pull requests from forks because members with write access have web preview
+      github.event_name != 'pull_request' || 
+      (github.event_name == pull_request && github.event.pull_request.head.repo.full_name != 'Kotlin/dokka')
```

Comment:
```suggestion
      (github.event_name == pull_request && github.repository != 'Kotlin/dokka')
```

Reply:
Addressed in https://github.com/Kotlin/dokka/pull/2951#discussion_r1160151905, let's keep the conversation there, and I'll change this if it needs to be changed